Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Android/JavaObjectsConverter.cs

```diff
@@ -12,7 +12,7 @@ public class JavaObjectsConverter
 {
     private static readonly AndroidJavaClass _errorAttachmentLogClass = new AndroidJavaClass("com.microsoft.appcenter.crashes.ingestion.models.ErrorAttachmentLog");
 
-    public static ErrorReport ConvertErrorReport(AndroidJavaObject errorReport)
+    public static ErrorReport ConvertErrorReport(AndroidJavaObject errorReport, bool isCrash = true)
```

Comment:
The value is always passed with true and we override to false after.
Also both constructors set it to true.
This is confusing.

Reply:
Oops, that should be taken out.